src/app.js

```diff
@@ -33,9 +40,14 @@ import './compressor';
 import './soundRecorder';
 import './peakDetect';
 import './gain';
-import './monosynth';
-import './polysynth';
+// import './monosynth';
+// import './polysynth';
```

Comment:
let's remove the code rather than comment it out

Reply:
Okayy

Reply:
un-resolving this because it doesn't appear to be resolved yet

Reply:
Sorry for that, let me make this comment out